examples/customresources/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: module buf
+
+buf:
+	cd ../../ && make tool-install
+	PATH="$(PATH):$(shell pwd)/../../bin/gotools/$(shell uname -s)-$(shell uname -m)" buf generate
```

Comment:
Not easily. I can't include that file, because all those paths are relative, and rely on pwd, So it ends up setting a path of examples/customresources/bin/gotools (which doesn't exist.) Trying to rewrite the definition to work from BOTH locations results in something even uglier and more complex.